src/InteractiveWindow/Editor/InteractiveWindow.csproj

```diff
@@ -121,6 +121,7 @@
     <Compile Include="Output\SortedSpans.cs" />
     <Compile Include="Output\ZoomableInlineAdornment.cs" />
     <Compile Include="PredefinedInteractiveTextViewRoles.cs" />
+    <Compile Include="InteractiveWindow.ReplSpanKind.cs" />
```

Comment:
I renamed it back from `ReplSpanKind` to 'InteractiveWindow.ReplSpanKind' after moved it from `EditorFeatures`, will manually fix it.